doc/how-to/remove_machine.md

```diff
@@ -10,8 +10,29 @@ Before removing the machine, ensure that there are no LXD instances or storage v
 See {ref}`how to remove instances <lxd:instances-manage-delete>` in the LXD documentation.
 See {doc}`how to remove OSDs <microceph:how-to/remove-disk>` in the MicroCeph documentation.
 
+```{note}
+If local storage was created, MicroCloud will have also added some default storage volumes that will need to be cleaned up:
+
+    lxc config unset storage.images_volume --target <name>
+    lxc config unset storage.backups_volume --target <name>
+    lxc storage volume delete local images --target <name>
+    lxc storage volume delete local backups --target <name>
+
+Any additional storage volumes belonging to this machine must also be deleted before removal without the `--force` flag.
+```
+
 If the machine is no longer reachable over the network, you can also add the `--force` flag to bypass removal restrictions and skip attempting to clean up the machine. Note that MicroCeph requires `--force` to be used if the remaining cluster size will be less than 3.
 
 ```{caution}
 Removing a cluster member with `--force` will not attempt to perform any clean-up of the removed machine. All services will need to be fully re-installed before they can be re-initialised. Resources allocated to the MicroCloud like disks and network interfaces may need to be re-initialised as well.
 ```
+
+### Reducing the cluster to 1 machine
+
+When shrinking the cluster down to 1 machine, you must also clean up the Ceph monmap before proceeding, even when using the `--force` flag.
+
+    sudo microceph.ceph mon remove <name>
```

Comment:
Is it worth having another test cover this bit?

Reply:
I've added a test case for shrinking down to 1 node, which requires this, as well as checking that we can't remove the node even with `--force` until it is removed from MicroCeph's monmap.

Additionally, it seems a lot of the remove tests were incorrectly ignoring expected errors, so those are fixed too.

As well, I noticed a small bug with `microcloud status` where uninitialized services were being ignored because the snap was installed. This is fixed by also reporting the service as missing if it is installed, but its cluster member count is 0.

Reply:
I just gave this a try by running `microcloud init` on a system with lxd 5.21 installed, and lxd 5.0 installed on a joiner. Indeed this is the error I see:
```
Error: Failed to join cluster: Failed to initialize member: Failed to initialize storage pools and networks: Failed to update local member project "default": Invalid project configuration key "features.networks.zones"
```

So it seems we eagerly try to update the projects table before doing any API/database version checking in `5.0`, thus getting back a non-descriptive error.

Still not sure why `snap info` listed `5.21`, but maybe I just failed to catch a `5.0` in there.